graphs/breathfirstsearch/breathfirstsearch.go

```diff
@@ -1,73 +1,31 @@
 package breathfirstsearch
```

Comment:
Add a package description so that it's easier to navigate it in the repository documentation.

```suggestion
// Package breadthfirstsearch <description goes here>
package breadthfirstsearch
```

Also just realised that the spelling of `breadthfirstsearch` was incorrect